pbf/migrations/0001_initial.py

```diff
@@ -4,6 +4,377 @@
+        # Minor,
+        [CARD_MINOR, "Hazards Spread Across The Island", 0, "Fire,Air,Earth,Plant"],
+        [CARD_MINOR, "Bats Scout For Raids By Darkness", 1, "Moon,Air,Animal"],
+        [CARD_MINOR, "Birds Cry Warning", 1, "Sun,Air,Animal"],
+        [CARD_MINOR, "Blood Draws Predators", 1, "Sun,Fire,Water,Animal"],
+        [CARD_MINOR, "Carapaced Land", 0, "Earth,Plant,Animal"],
+        [CARD_MINOR, "Call To Guard", 0, "Sun,Air,Earth"],
+        [CARD_MINOR, "Desiccating Winds", 1, "Fire,Air,Earth"],
+        [CARD_MINOR, "Dry Wood Explodes In Smoldering Splinters", 1, "Fire,Air,Plant"],
+        [CARD_MINOR, "Gift of Nature's Connection", 0, ""],
+        [CARD_MINOR, "Entrap The Forces Of Corruption", 1, "Earth,Plant,Animal"],
+        [CARD_MINOR, "Domesticated Animals Go Berserk", 1, "Moon,Fire,Animal"],
+        [CARD_MINOR, "Dire Metamorphosis", 1, "Moon,Air,Earth,Animal"],
+        [CARD_MINOR, "Skies Herald The Season Of Return", 1, "Sun,Moon,Plant,Animal"],
+        [CARD_MINOR, "Gift of Twinned Days", 1, "Sun,Moon"],
+        [CARD_MINOR, "Haunted By Primal Memories", 1, "Moon,Air,Earth"],
+        [CARD_MINOR, "Like Calls to Like", 1, "Sun,Water,Plant"],
+        [CARD_MINOR, "Unquenchable Flames", 1, "Moon,Fire,Earth"],
+        [CARD_MINOR, "Mesmerized Tranquility", 0, "Water,Earth,Animal"],
+        [CARD_MINOR, "Territorial Strife", 0, "Sun,Fire,Animal"],
+        [CARD_MINOR, "Renewing Boon", 1, "Sun,Earth,Plant"],
+        [CARD_MINOR, "Scream Disease Into The Wind", 1, "Air,Water,Animal"],
+        [CARD_MINOR, "Sear Anger Into The Wild Lands", 0, "Sun,Fire,Plant"],
+        [CARD_MINOR, "Set Them On An Ever-Twisting Trail", 1, "Air,Plant,Animal"],
+        [CARD_MINOR, "Strong And Constant Currents", 0, "Sun,Water,Earth"],
+        [CARD_MINOR, "Sucking Ooze", 0, "Moon,Water,Earth"],
+        [CARD_MINOR, "Sunset's Fire Flows Across The Land", 1, "Sun,Moon,Fire,Water"],
+        [CARD_MINOR, "Terror Turns To Madness", 0, "Moon,Air,Water"],
+        [CARD_MINOR, "The Shore Seethes With Hatred", 1, "Fire,Water,Earth,Plant"],
+        [CARD_MINOR, "Thriving Chokefungus", 1, "Moon,Water,Plant"],
+        [CARD_MINOR, "Treacherous Waterways", 0, "Fire,Water,Earth"],
+        [CARD_MINOR, "Flow Downriver, Blow Downwind", 0, "Air,Water,Plant"],
+        [CARD_MINOR, "Weep For What Is Lost", 0, "Fire,Water,Animal"],
+        [CARD_MINOR, "Favor Of The Sun And Star-Lit Dark", 1, "Sun,Moon,Plant"],
+)
+
+def load_initial_content(apps,schema_editor):
+    Spirit = apps.get_model("pbf", "Spirit")
+    Card = apps.get_model("pbf", "Card")
+
+    for s in spirits:
+        Spirit(name=s[0]).save()
+
+    # Unique card definition format:
+    #  Spirit            Card name      Cost   Elements 
+    # ["Lightning", "Shatter Homesteads", 2, "Fire,Air"],
+    for c in uniques:
+        Card(name=c[1], type=CARD_UNIQUE, spirit=Spirit.objects.get(name=c[0]), cost=c[2], elements=c[3]).save()
```

Comment:
idea:

```suggestion
    for (spirit, name, cost, elements) in uniques:
        Card(name=name, type=CARD_UNIQUE, spirit=Spirit.objects.get(name=spirit), cost=cost, elements=elements).save()
```

naming each element inside the `for` makes the code self-documenting. perhaps this is even sufficiently clear enough that the comment is not needed?

Reply:
Oh, yeah I like that a lot! I always forget this sort of tuple syntax exists.